content/people/peymant.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Peyman Tehrani
+role: student-phd
+status: Ph.D. Student
+yearjoined: 2017
+portrait: peymant.jpg
+badge: peymant-badge.jpg
+interests:
+  - Resource allocation and optimization algorithms 
+  - Wireless communications and networks
+  - Machine learning and Reinforcement learning
+  - Internet of things and edge computing
+  - Unmanned aerial vehicles
+education:
+  - degree: M.S. in Electerical Engineering-Communication Systems
+    institution: University of Tehran, Iran
+    year: 2016
+  - degree: B.S. in Electrical Engineering
+    institution: University of Tehran, Iran
+    year: 2013
+email: peymant@uci.edu
+websites:
+    - title: LinkedIn
+      url: www.linkedin.com/in/peyman-tehrani-939195a3/
+    - title: Google Scholar
+      url: https://scholar.google.com/citations?user 1hbKmhYAAAAJ&hl=en

```

Comment:
The link is broken. It looks like the space after `?user` should be changed to `=`.